src/Uno.UI/Uno.UI.Reference.csproj

```diff
@@ -140,6 +140,5 @@
 	</ItemGroup>
 
 	<Import Project="MixinGeneration.targets" />
-	<Import Project="XamlMerge.targets" />
```

Comment:
> This is the right fix, as the `xaml` files are not automatically included in reference build (`EnableAutomaticXamlPageInclusion` is `false`)

Let's make sure it won't cause issues on platforms that rely on the presence of the `GlobalStaticResources` on the reference assemblies (it may only appear when running from the uno.sdk, not inside the samples app).

Reply:
I've tested Gallery with the CI artifacts and it seems fine without a GlobalStaticResources for the reference project.